WeakAurasOptions/RegionOptions/Model.lua

```diff
@@ -1,4 +1,5 @@
 local L = WeakAuras.L;
+if WeakAuras.IsClassic then return end -- Models disabled for classic
```

Comment:
@Stanzilla here too, we might save a bit on classic package size if we just don't ship model files